lib/sidekiq.rb

```diff
@@ -97,7 +97,7 @@ def self.server_middleware
   end
 
   def self.default_worker_options=(hash)
-    @default_worker_options = default_worker_options.merge(hash)
+    @default_worker_options = default_worker_options.merge(hash).stringify_keys!
```

Comment:
I believe that stringify_keys! is Rails-specific?  It needs to work with plain Ruby.